jax/_src/scipy/linalg.py

```diff
@@ -26,6 +26,7 @@
 from jax._src.numpy.util import _wraps
 from jax._src.numpy import lax_numpy as jnp
 from jax._src.numpy import linalg as np_linalg
+from jax._src.scipy.util import _no_chkfinite_doc, _no_overwrite_and_chkfinite_doc, _no_overwrite_and_chkfinite_skip_params
```

Comment:
Rather than defining the shared docstring in a separate module, I'd define them as global variables at the top of this file.

And rather than defining the list of parameters in a global variable, I'd just list them in each case they're used. The principle behind this thought is that we should avoid action at a distance unless there are good reasons to do otherwise; in this case the utilities are so short that I think inlining them with some repetition is preferable to having to click to another source file to find out what things are.

Reply:
Just to underscore this, the variable name is as long as the list of parameters it refers to!
```
_no_overwrite_and_chkfinite_skip_params
('overwrite_a', 'overwrite_b', 'check_finite')
```

Reply:
Nice insight, I will follow your advice after I wake up, I have to go to bed now.